cli/internal/context/context.go

```diff
@@ -147,13 +146,13 @@ func WithGraph(rootpath string, config *config.Config) Option {
 			return fmt.Errorf("could not resolve workspaces: %w", err)
 		}
 
-		spaces, err := c.Backend.GetWorkspaceGlobs(rootpath)
+		spaces, err := c.PackageManager.GetWorkspaceGlobs(rootpath)
 
 		if err != nil {
 			return fmt.Errorf("could not detect workspaces: %w", err)
 		}
 
-		globalHash, err := calculateGlobalHash(rootpath, config.RootPackageJSON, config.TurboConfigJSON.GlobalDependencies, c.Backend, config.Logger, os.Environ())
+		globalHash, _ := calculateGlobalHash(rootpath, config.RootPackageJSON, config.TurboConfigJSON.GlobalDependencies, c.PackageManager, config.Logger, os.Environ())
```

Comment:
I think this error was previously unintentionally ignored. We should handle it, probably the classic 
```go
if err != nil { 
  return err
}
```

will be fine here.

Reply:
I was gonna let your PR implement it (https://github.com/vercel/turborepo/pull/1062/files#diff-44b5ffa1d43d53679fd2605faaf19a651ec0be80bbe451410199d9f335274b37R158-R160)

But I'll pull that over to this one.